user-guide/src/basic.md

```diff
@@ -3,14 +3,64 @@
 When the Prusti Assistant extension is active, Rust files can be verified in one of the following ways:
 
 - By clicking the "Verify with Prusti" button in the status bar.
+- By opening the [Command Palette](https://code.visualstudio.com/docs/getstarted/userinterface#_command-palette) running the command `Prusti: save and verify this file`
 - By saving a Rust document, if "Verify on save" is enabled.
 - By opening a Rust document, if "Verify on open" is enabled.
 
 See the [following chapter](verify/summary.md) for a list of verification features available in Prusti.
 
-## Example
+## Introductory Example
 
-The specification for a `max` function (returning the greater of its two inputs) could be:
+Let us verify that the function `max` below, which takes two integers and returns the larger one, is implemented correctly.
+
+```rust
+fn max(a: i32, b: i32) -> i32 {
+    if a > b {
+        a
+    } else {
+        b
+    }
+}
+```
+
+When pasting the above code into a Rust file and then verifying it with Prusti (as outlined at the top of the page), Prusti should report that verification succeeded.
+This tells us that
+
+1. the file consists of valid Rust code that can be compiled successfuly, and
+2. no execution reaches an explicit call of the [`panic!()`](https://doc.rust-lang.org/std/macro.panic.html) macro.
```

Comment:
I guess "explicit" is misleading, since a panic can be caused e.g. by an assertion failure.